app/src/main/java/com/example/myapplication/ThirdFragment.java

```diff
@@ -36,35 +43,150 @@ public class ThirdFragment extends Fragment {
 
 
     private FragmentThirdBinding binding;
+    ViewGroup v = null;
 
 
     @Override
     public View onCreateView(
             LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState
     ){
         binding = FragmentThirdBinding.inflate(inflater, container, false);
+        v = container;
+        ColorStateList purple = ColorStateList.valueOf(Color.parseColor(("#6750A3")));
+        ColorStateList blue = ColorStateList.valueOf((Color.parseColor("#73C2F0")));
+        if (binding.switch1.isChecked()){
+            binding.switch1.setThumbTintList(purple);
+            binding.switch1.setTrackTintList(purple);
+        } else {
+            binding.switch1.setThumbTintList(blue);
+            binding.switch1.setTrackTintList(blue);
+        }
+        binding.switch2.setChecked(MainActivity.defense);
+        if (binding.switch2.isChecked()){
+            binding.switch2.setThumbTintList(blue);
+            binding.switch2.setTrackTintList(blue);
+        } else {
+            binding.switch2.setThumbTintList(blue);
```

Comment:
Should the color be blue both if `binding.switch2.isChecked()` and if it is not checked?

Reply:
fixed